docs/guides/data-loading.md

```diff
@@ -357,15 +357,15 @@ export default function ProductFilters() {
 
 If the user only has one selected:
 
-- [x] Nike
-- [ ] Adidas
+- \[x] Nike
+- \[ ] Adidas
```

Comment:
Same here 🤔

Reply:
It seems like this is a `remark` problem.

@wooorm Is this expected default behavior? If so, How could we make sure the backslash (`\`) is removed?

Reply:
Us remark-gfm to enable gfm in markdown